gecoscc/templates/bb-printer.html

```diff
@@ -101,6 +101,47 @@
                     </div>
                 </div>
             </div>
+<<<<<<< HEAD
```

Comment:
"HEAD" tags are added by git in order to mark conflicts, but must be deleted before commiting changes.